conda/models/records.py

```diff
@@ -262,6 +263,11 @@ class PackageRecord(DictSafeMixin, Entity):
         default=None, required=False, nullable=True, default_in_dump=False
     )
 
+    @property
+    def is_prerelease(self) -> bool:
+        """True if package has prerelase version ending with an alphabetic tag other than 'post'"""
+        return bool(version_prerelease_re.match(self.version))
+
```

Comment:
I'd rather see this as a method that takes the optional kwarg mentioned earlier for the context configuration. E.g. `openssl <3` should never make this return True.

Reply:
ok